tests/e2e/verification/test_get_report_using_id.py

```diff
@@ -0,0 +1,23 @@
+from sinch.domains.verification.models.responses import GetVerificationByIdResponse
+
+
+def test_get_report_verification_using_id(
+    sinch_client_sync,
+    phone_number,
+    verification_id
+):
+    verification_response = sinch_client_sync.verification.verification_status.get_by_id(
+        id=verification_id
+    )
+    assert isinstance(verification_response, GetVerificationByIdResponse)
```

Comment:
To really ensure against regression we should test field values and not only response instance type

Reply:
Request data validation is being handled by Wiremock (and recored fixtures).
When regression happens, Wiremock will validate that data and return error to the test suite.